.travis/maximize_fuzz_corpus.sh

```diff
@@ -17,7 +17,7 @@ set -e
 
 ALL_FUZZ_TEST_FILES=""
 
-for dir in $( ls ./tests/fuzz/corpus ); do
+for dir in $( find ./tests/fuzz/corpus -name "s2n_*_test" | sed 's/.*\///g' ); do
```

Comment:
I think using `basename` instead of `sed` is a lot clearer that we're trying to strip off the leading path to the directory. I think we should replace it with: `find ./tests/fuzz/corpus -name "s2n_*_test" -type d -exec basename {} \;`

Reply:
Sure. Added at https://github.com/thereadonly/s2n/commit/e42a7c8462697244fb0b7d065bfc4455b023357d